backend/src/HttpExceptionFilter.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  Catch,
+  ArgumentsHost,
+  HttpException,
+  HttpStatus,
+} from '@nestjs/common';
+import { BaseExceptionFilter } from '@nestjs/core';
+
+import { Request, Response } from 'express';
+
+@Catch(Error, HttpException)
+export class HttpExceptionFilter extends BaseExceptionFilter {
```

Comment:
not sure what this class is used to,
maybe add doc (let chat GPT do that for you)